doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -1304,6 +1304,26 @@ These field section entries are described in :numref:`Table %s <FieldDescription
 
 Comments can be added to the diag_table using the hash symbol (``#``).
 
+
+Each WM component has its own ``diag_table`` with associated variables. :numref:`Table %s <diag-table-options>` contains links to the full set of options for each WM component.
+
+.. _diag-table-options:
+
+.. list-table:: * *diag_table* Options for WM Components*
+   :widths: 16 24 
```

Comment:
```suggestion
   :widths: 16 24 24
```
Since there are 3 columns, we need three widths or the docs won't build the table.